doc/packer.txt

```diff
@@ -96,22 +96,28 @@ Then you can write your plugin specification in Lua, e.g. (in `~/.config/nvim/lu
 
 `packer` provides the following commands: *packer-default-commands* *packer-commands*
 
-`PackerClean`                                   *packer-commands-clean*
+`Packer clean`                                          *packer-commands-clean*
 Remove any disabled or unused plugins.
 
-`PackerInstall`                                 *packer-commands-install*
+`Packer install`                                      *packer-commands-install*
 Clean, then install missing plugins.
 
-`PackerUpdate`                                  *packer-commands-update*
+`Packer update`                                        *packer-commands-update*
 Update installed plugins.
 Supports the `--preview` flag as an optional first argument to preview
 updates.
 
-`PackerSync`                                    *packer-commands-sync*
+`Packer sync`                                            *packer-commands-sync*
 Clean, then update and install plugins.
 Supports the `--preview` flag as an optional first argument to preview
 updates.
 
+`Packer lock`                                            *packer-commands-sync*
+TODO
+
+`Packer restore`                                      *packer-commands-restore*
```

Comment:
Nitpick: should this be `restore` or something like `load`?

Reply:
`load` surely means load a plugin right. This action is checking out older revisions of plugins to pinned versions, so either `restore` or `rollback`.

Reply:
To me, `load` means to load the state described by a lockfile. `restore` would be to go back to a previous state after an update, which is a bit different.

Maybe `load-lockfile`? We can also workshop the names later.